src/understanding-deis/concepts.md

```diff
@@ -11,27 +11,52 @@ We consider it an invaluable synthesis of much experience with
 software-as-a-service apps in the wild, especially on the
 Heroku platform.
 
-Deis is designed to run applications that adhere to [Twelve-Factor App][]
+Workflow is designed to run applications that adhere to [Twelve-Factor App][]
```

Comment:
I think "Deis the product" is still the name people come to recognize here. It's like calling the latest Porsche 911 the 991.2 because that's what the engineers call it. Nobody but the engineers and people who are "in the know" know about the internal name change from "Controller" to "Workflow", much like the model change from 991 to 991.2.